### DISCUSSION TITLE
Azure OpenAI integration

### DISCUSSION THREAD

**Discussion by tiangao88** [Yue-Yang/ChatGPT-Siri#118] [Ideas]
Could we have a version that uses Azure OpenAI API ?
Thanks!

**Comment by FedorIvachev** [comment]
Hi, you can check my fork (use your API key, endpoint and deployment name).

> Could we have a version that uses Azure OpenAI API ?
> Thanks!

https://github.com/FedorIvachev/AzureGPT-Siri
